tests/pyccel/test_pyccel_make.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+from pathlib import Path
+import shutil
+import subprocess
+import sys
+import pytest
+from test_pyccel import get_python_output, compare_pyth_fort_output
+
+current_folder = Path(__file__).parent
+
+@pytest.fixture( params=[
+        pytest.param("cmake", marks = pytest.mark.cmake),
+        pytest.param("meson", marks = pytest.mark.meson),
+    ],
+)
+def build_system(request):
+    """
+    Test with the chosen build system.
+    """
+    return request.param
+
+#------------------------------------------------------------------------------
+def pyccel_make_test(main_file, folder, language, build_system, args, output_dtype = float):
+    """
+    Test the pyccel-make command.
+
+    Parameters
+    ----------
+    main_file : str
+        The file containing the `__main__` implementation.
+    folder : Path
+        The folder in which the test is found.
+    build_system : str
+        The build system that should be used for compilation.
```

Comment:
The `language` parameter seems to be missing. Further, only two values are accepted for the `build_system` argument:

```suggestion
    language : {'c', 'fortran', 'python'}
        The language used for translation.
    build_system : {'cmake', 'meson'}
        The build system that should be used for compilation.
```